src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -0,0 +1,122 @@
+                     username=user, password=test_password)
+    (result, result1, exit_status) = ssh1.exec_command(f"ipa  "
+                                                       f"subid-find"
+                                                       f"  --owner  "
+                                                       f"{user}")
+    user_details = result1.readlines()
+    global uid_start, uid_range, gid_start, gid_range
+    uid_start = int(user_details[5].split(': ')[1].split('\n')[0])
+    uid_range = int(user_details[6].split(': ')[1].split('\n')[0])
+    gid_start = int(user_details[7].split(': ')[1].split('\n')[0])
+    gid_range = int(user_details[8].split(': ')[1].split('\n')[0])
+    ssh1.close()
+
+
+@pytest.mark.usefixtures('environment_setup',
+                         'subid_generate',
+                         'bkp_cnfig_for_subid_files')
+@pytest.mark.tier1
+class TestSubid(object):
+    """
+    This is for ipa bugs automation
+    """
+    def test_subid_feature(self, multihost):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 50bcdc28-00c8-11ec-bef4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Test newuidmap command
+            2. Test newgidmap command
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        ipa_subid_find(multihost)
+        ssh1 = SSHClient(multihost.client[0].ip,
+                         username=user, password=test_password)
+        (results1, results2, results3) = ssh1.exec_command("unshare"
+                                                           " -U bash"
+                                                           " -c 'echo $$"
+                                                           ">/tmp/unshare.pid;"
+                                                           "sleep 1000'")
+        time.sleep(2)
+        proces_id = int(execute_cmd(multihost,
+                                    "cat "
+                                    "/tmp/unshare.pid").stdout_text.strip())
+        uid = 0
+        count = 1
+        (result, result1, exit_status) = ssh1.exec_command(f"newuidmap"
+                                                           f" {proces_id} "
+                                                           f"{uid} "
+                                                           f"{uid_start} "
+                                                           f"{count}")
+        assert exit_status.readlines() == []
+        (result, result1, exit_status) = ssh1.exec_command(f"newgidmap "
+                                                           f"{proces_id} "
+                                                           f"{uid} "
+                                                           f"{gid_start} "
+                                                           f"{count}")
+        assert exit_status.readlines() == []
```

Comment:
https://github.com/SSSD/sssd/blob/4b7b6fa70399654b8a6d2f691be832c0680ee1fb/src/tests/multihost/sssd/testlib/common/utils.py#L1660

`exist_status` is http://docs.paramiko.org/en/stable/api/channel.html?highlight=recv_exit_status#paramiko.channel.Channel.recv_exit_status : `Returns: | the exit code (as an int) of the process on the server.`

So it looks it should be `0`, not empty..?

Reply:
Well, @aborah-sudo says `exit_status` contains text with error message (`stderr`..?)
Not sure I understand this, but ok.